src/Microsoft.Health.Dicom.Client/DicomWebClient.ExtendedQueryTag.cs

```diff
@@ -91,5 +91,25 @@ public async Task<DicomWebResponse<IEnumerable<ExtendedQueryTagError>>> GetExten
             await EnsureSuccessStatusCodeAsync(response).ConfigureAwait(false);
             return new DicomWebResponse<IEnumerable<ExtendedQueryTagError>>(response, ValueFactory<IEnumerable<ExtendedQueryTagError>>);
         }
+
+        public async Task<DicomWebResponse<GetExtendedQueryTagEntry>> UpdateExtendedQueryTagAsync(string tagPath, UpdateExtendedQueryTagEntry newValue, CancellationToken cancellationToken)
+        {
+            EnsureArg.IsNotNullOrWhiteSpace(tagPath, nameof(tagPath));
+            EnsureArg.IsNotNull(newValue, nameof(newValue));
+            EnsureArg.EnumIsDefined(newValue.QueryStatus, nameof(newValue.QueryStatus));
+            string jsonString = JsonConvert.SerializeObject(newValue);
+            var uri = new Uri($"/{_apiVersion}{DicomWebConstants.BaseExtendedQueryTagUri}", UriKind.Relative);
+
+            using var request = new HttpRequestMessage(HttpMethod.Patch, uri);
+            {
+                request.Content = new StringContent(jsonString);
```

Comment:
I think you can use `JsonContent` instead

Reply:
`JsonContent` use `System.Text.Json` for serialization,  as we already use `Newtonsoft.Json`,  better to use same.